src/routes/(signedIn)/(emailVerified)/registrations/+page.svelte

```diff
@@ -134,6 +134,14 @@
       goto(`?${base.toString()}`)
     }
   }
+  function bypassAgeLimits(id:string, bypassAgeLimits:boolean) {
+    getDoc(doc(db, 'registrationsSpring24', id)).then((applicationSnapshot) => {
+      if (applicationSnapshot.exists()) {
+        console.log(applicationSnapshot.data().agreements.bypassAgeLimits)
+        updateDoc(doc(db, 'registrationsSpring24', id), {'agreements.bypassAgeLimits': !applicationSnapshot.data().agreements.bypassAgeLimits});
```

Comment:
shouldn't this be set to registration.values.agreements.bypassAgeLimits rather than just toggling it